### PR TITLE
Update tools.md

### PR DESCRIPTION
**Pull request by kathryn-ods**
adding guidance on installing and building the docs in the development environment as per #509

### CODE REVIEW

**Comment by kathryn-ods** [comment]
@kd-ods please will you check over the guidance I have written here?

File: tools.md [hunk 1]
```diff
@@ -4,6 +4,44 @@
 
 BODS docs are published on [readthedocs](https://standard.openownership.org).
 
+## Building the docs using the development environment
```
Comment (kd-ods) [inline tools.md:7]:
```suggestion
## The development environment for BODS
```

File: tools.md [hunk 2]
```diff
@@ -4,6 +4,44 @@
 
 BODS docs are published on [readthedocs](https://standard.openownership.org).
 
+## Building the docs using the development environment 
+### Installing the development environment
```
Comment (kd-ods) [inline tools.md:8]:
```suggestion

Using the development environment means that you needn't maintain relevant python versions and libraries on your system. Plus the environment can be updated to support common schema development practices.
```

File: tools.md [hunk 3]
```diff
@@ -4,6 +4,44 @@
...
+Install the ODSC development environment by following the instructions at https://github.com/OpenDataServices/development-environment
+
+The setup process guides you through several options
+- For 'Please select a container management engine to use' either will work (use Docker if you don't have a preference) 
+- For 'Do you want to mirror the environment's working directory locally?' select 'yes'
+- Select your preferred text editor to be installed (use Pycharm if you don't have a preference)
+- Select 'Beneficial Ownership Data Standard (BODS)' to be installed in the environment
+- Do you want to setup a new development branch? Select 'no' unless you want to set one up
+
+### Building the docs
```
Comment (kd-ods) [inline tools.md:18]:
```suggestion
### Building the docs within the dev environment
```

File: tools.md [hunk 4]
```diff
@@ -4,6 +4,44 @@
...
+Open a terminal and navigate to the development environment 
+
+``cd development-environment``
+
+connect to the environment 
+
+``bin/connect``
+
+navigate to the data standard
```
Comment (kd-ods) [inline tools.md:27]:
```suggestion
navigate to the directory containing the data standard, e.g.:
```

**Review by kd-ods — CHANGES_REQUESTED** [review]
I've suggested a few things to flip the framing. (Since the dev environment is the tool, and building the docs is only one thing you can do using it.) See what you think.

**Comment by kathryn-ods** [comment]
Thanks @kd-ods accepted all those and will merge :)

**Pull request merged**